Spigot-Server-Patches/0424-Call-WhitelistToggleEvent-when-whitelist-is-toggled.patch

```diff
@@ -0,0 +1,21 @@
+From f299fb452e762d8c151e19ac018f541861085903 Mon Sep 17 00:00:00 2001
+From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
+Date: Wed, 13 Mar 2019 20:08:09 +0200
+Subject: [PATCH] Call WhitelistToggleEvent when whitelist is toggled
+
+
+diff --git a/src/main/java/net/minecraft/server/PlayerList.java b/src/main/java/net/minecraft/server/PlayerList.java
+index 408c382d..b1630137 100644
+--- a/src/main/java/net/minecraft/server/PlayerList.java
++++ b/src/main/java/net/minecraft/server/PlayerList.java
+@@ -1329,6 +1329,7 @@ public abstract class PlayerList {
+     }
+ 
+     public void setHasWhitelist(boolean flag) {
++        new com.destroystokyo.paper.event.server.WhitelistToggleEvent(flag).callEvent();
```

Comment:
Are we sure nothing else calls whitelist.setEnabled() skipping this method?

Reply:
As much as I went through with IntelliJ then nothing else calls `PlayerList#whitelist#setEnabled(boolean)`.